packages.json

```diff
@@ -2,6 +2,12 @@
   "packages": {
     "plugins": [
       {
+      "name": "Ligature Xcode plug-in",
```

Comment:
This needs to match the `.xodeproj` file, as per the README. So it should be: `LigatureXcodePlugin`